docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -62,25 +41,39 @@ openstack image list -c ID -c Status |grep -E '\| .+ing \|'
 Also collect the services topology specific configuration before stopping services required to gather it live. You will need it to compare it with the post-adoption values later on. For more information, see xref:pulling-the-openstack-configuration_{context}[Pulling the OpenStack configuration].
```

Comment:
This link doesn't work in the preview

Reply:
I will update in a separate PR when I eventually rename the "Pulling configuration" module.